boards/nucleo-wl55jc/include/periph_conf.h

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2021 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nucleo-wl55jc
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the nucleo-wl55jc board
+ *
+ * @author      Akshai M <akshai.m@fu-berlin.de>
+ * @author      Hauke Petersen <devel@haukepetersen.de>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+/* Add specific clock configuration (HSE, LSE) for this board here */
+#ifndef CONFIG_BOARD_HAS_LSE
+#define CONFIG_BOARD_HAS_LSE            1
+#endif
+
+/* This board provides a 32MHz HSE oscillator  */
+#ifndef CONFIG_BOARD_HAS_HSE
+#define CONFIG_BOARD_HAS_HSE            1
+#endif
+
+#define CLOCK_HSE                       MHZ(32)
+
+/* VCO output frequency ((PLL input clock frequency / PLLM ) x PLLN ) must be
+   between 96 and 344 MHz. PLLN can have values <=127 & >=6 */
+/* use a core clock of 48MHz and run APBx buses at the same speed */
+#define CONFIG_CLOCK_PLL_N              18
```

Comment:
The `CPU_FAM_STM32WB` defines some things like `CLOCK_CORECLOCK_MAX` in `cpu/stm32/include/clk/l4l5wb/cfg_clock_default.h`, can you adapt and/or adapt the equivalent? Also since you are using that file already could you maybe move this definition there? That was its already `infdef` guarded.